src/components/DialogBox/styles.scss

```diff
@@ -17,6 +15,17 @@
   z-index: 1300;
   inset: 0px;
 
+  &--skrim {
+    background: linear-gradient(180deg, rgba($tw-clr-base-gray-025, $tw-opcty-base-060), rgba(#EFF5EF, $tw-opcty-base-060));
```

Comment:
Asked for a token for `#EFF5EF`, we didn't think we'd need it at first, but it would be helpful here.

Reply:
should we be using semantic tokens here (for the ones with tokens)?
gives designers more control

Reply:
was told that base colors should be the way to go for this background gradient

Reply:
fair enought